api/v1beta1/event.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.31.0
-// 	protoc        v4.23.4
+// 	protoc-gen-go v1.27.1
+// 	protoc        v3.19.4
```

Comment:
@pdeziel would you mind updating your `protoc` and `proto-gen-go` please? 

I think you can do this with:

```
$ brew upgrade protobuf
$ go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
$ go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
```

I'm on:

```
protoc --version
libprotoc 23.4
```

And I think the protoc-gen-go v1.31 and protoc-gen-go-grpc v1.3 have some interesting features that we can take advantage of.